helpers/typeof.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+const common = require('./lib/common.js');
+
+const factory = globals => {
+    return function(data) {
+        data = common.unwrapIfSafeString(globals.handlebars, data);
```

Comment:
Helpers should always handle the Options object, otherwise there can be unexpected results. In this case, forgetting to pass an argument will lead to a return value of `object` rather than an error during development.
```suggestion
      return function(data) {
            arguments.pop(); // remove Options
            if (arguments.length !== 1) {
                throw new ValidationError("Handlebars helper 'typeof' requires exactly one argument.");
            }
            data = common.unwrapIfSafeString(globals.handlebars, data);
```